replset-consistency/README.md

```diff
@@ -142,6 +142,8 @@ It is important to check on all nodes as the contents on each node can vary, and
 
 If `local.system.healthlog` does not contain a "dbCheckStart" document for the collection you're running `dbCheck` on, `local.system.healthlog` has rolled over and subsequent scripts will not have complete information. 
 
+If `dbCheck` is run using `dbcheck.js`, you may optionally specify an `authInfo` object, with the user created above, to automatically check if the healthlog has rolled over. Omitting this object will not accurately check if the healthlog has rolled over on each secondary.
```

Comment:
Please go stronger: "This option is required for an accurate check on the completeness of results from all replica set nodes."

Reply:
"After execution, carefully consider primaryCount and secondaryCount in the output to ensure they are equal", or something?